### ISSUE TITLE
bug? test time: only constant gray images

### ISSUE THREAD

**Issue by AlexBinder** [artcg/BEGAN#13]
Hi Artur,

there were some minor fixed necessary

add-epochs=0 ->  --add-epochs=0
--train=False --> --train=0
also save the images as png, not jpg
but then still the images were constant grey, as an
            plt.imshow(im_to_save)
            plt.show()
could verify

values were all around 0.49...0.50, normalizing them to 0...1 gives only color clouds

**Comment by AlexBinder** [comment]
tensorflow 1.2.0 rc1

**Comment by anastasiadk** [comment]
I have the same issue as well. I changed a couple of things (removed non-linearities, subtracted 0.5), as suggested in this [issue](https://github.com/carpedm20/BEGAN-tensorflow/issues/3) related to [carpedm20/BEGAN-tensorflow](https://github.com/carpedm20/BEGAN-tensorflow), but still it doesn't converge and output at test time is all gray images, as AlexBinder stated above.
Any idea about why this happens?
Thanks in advance.

**Comment by ChesterAiGo** [comment]
I got the same problem...testing using the original CelebA dataset now..have you guys solved the problem? It might come from the usage of our custom dataset i'd say. 

EDIT: Tested on a subset of CelebA (2600 images), for 10 epochs (I did not expect any outstanding results by using 10 epochs, but at least it should not be all grey...) with all default configs and it still outputs grey images...weird

**Comment by DLjiaerlin** [comment]
I am suffer from some issues when trained a model and want to generate some samples simply run;
when I execute "python main.py --start-epoch=N add-epochs=0 --train=False"this line,some error appear.
can you share execute comand line?
where N is the checkpoint ,I have a little confused what it is a num or produced model? i have produced the model eg.BEGAN_64_64_0005.tfmod.data-00000-of-00001.I try to set "N" as "1or5",and "-train=False"when execute is a error,I replaced it with "train=0 or train=1",only this way,the code when execute is occor errors.
can you give some guide about you generate the samples?
thanks again!

**Comment by artcg** [comment]
Closing this issue which turned out to be a tensorflow version issue
